src/App.jsx

```diff
@@ -1,12 +1,16 @@
 import { React, useEffect, useState } from 'react';
 import axios from 'axios';
 import WeatherProvider from './components/WeatherProvider';
+import { useTheme } from './components/ThemeProvider';
+import ToggleButton from './components/ToggleButton';
 
 const geocodingURI = 'http://api.openweathermap.org/geo/1.0/direct?q=';
 const weatherApiKey = process.env.REACT_APP_WEATHER_API_KEY;
 
 function App() {
   const [geoData, setGeoData] = useState({});
+  const { theme } = useTheme();
+  const appClass = `App container-fluid ${theme}`;
```

Comment:
Can you clarify these two lines for me. useTheme is a function that returns a useContext(ThemeContext) correct? I'm confused where theme is coming from and why it's being destructured? As for the appClass variable, I believe that can be embeded directly into the div className using curly braces {} so a variable is not needed? However, I don't think I fully understand the context so I may definitely be wrong.

Reply:
I understand where theme is coming from now, just had to understand how the context provider works! Ignore that :).
Still unsure about appClass though

Reply:
That's confusing, you're right. The `appClass` is unnecessary and I moved it to the inline `className=`.